docs/web_scraping_101/anti_scraping_techniques.md

```diff
@@ -74,15 +74,15 @@ To make things complicated, websites often employ various scraping protection co
 
 ## [](#reducing-blocking-with-shared-ip-address-emulation) Reducing blocking with shared IP address emulation
 
-Even though IP address rotation and emulation of browser HTTP signatures can get you far, there is a chance that for large scale crawls you’ll start getting blocked. Of course, you can always add more proxies, but that can become overly expensive.
+IP address rotation and emulation of browser HTTP signatures can be effective for many webscraping tasks, but large-scale crawls will get blocked. Using more proxies is a solution to this, but that can be expensive.
 
-Shared IP address emulation can dramatically extend the effectiveness of your scraping and multiply the number of pages that you can fetch. The technique relies on websites knowing that many different users can be behind a single IP address.
+Shared IP address emulation can dramatically extend the effectiveness of scraping and multiply the number of pages that can be fetched. The technique relies on websites knowing that many different users can be behind a single IP address.
 
-For example, requests from mobile devices are usually only routed through a handful of IP addresses, while users behind a single corporate firewall might all have a single IP address. By emulating and managing these user sessions per IP address, it is possible to trick websites into limiting how aggressive their blocking is.
+For example, requests from mobile devices are usually only routed through a handful of IP addresses, while users behind a single corporate firewall might all have a single IP address. By emulating and managing these user sessions per IP address, we found that it is possible to prevent websites from aggressive blocking.
 
-To make it work, you need to make sure that a single user session is always routed via the same IP address. A website can identify such user sessions based on cookies, authentication tokens or a browser HTTP signature/fingerprint.
+To make it work, a single user session has to always be routed via the same IP address. A website can identify such user sessions based on cookies, authentication tokens or a browser HTTP signature/fingerprint.
 
-The [Apify SDK](https://sdk.apify.com)'s [`SessionPool`](https://sdk.apify.com/docs/api/session-pool#docsNav) class lets you do just that. It is easy to add to other Apify tools such as [actors]({{@link actors.md}}) or [proxy]({{@link proxy.md}}) but also works outside of the Apify ecosystem.
+Our research into this was assisted by the [Apify SDK](https://sdk.apify.com)'s [`SessionPool`](https://sdk.apify.com/docs/api/session-pool#docsNav) class. This can also be added to other Apify tools such as [actors]({{@link actors.md}}) or [proxy]({{@link proxy.md}}) but it also works outside of the Apify ecosystem.
```

Comment:
Could "This can also be added" be "This can be added" to avoid a double "also" later in the sentence?

Reply:
Absolutely, thanks for catching this too!